src/storage/txn/actions/commit.rs

```diff
@@ -62,19 +64,36 @@ pub fn commit<S: Snapshot>(
         _ => {
```

Comment:
It is also unexpected to see a pessimistic type lock with the same `start_ts` committing secondary locks, the `mvcc` information is needed to be printed here too.

Reply:
OK

Reply:
Seems it is an expected case which may happen sometimes?

https://github.com/tikv/tikv/blob/1f44fd9a8e59d06c39834cfa0bd4d47e2c3c2959/src/storage/txn/actions/commit.rs#L44-L57

Reply:
Just print mvcc without returning an error in this case